Solutions/Cloudflare/Data Connectors/azuredeploy_Cloudflare_API_FunctionApp.json

```diff
@@ -168,7 +168,9 @@
                         "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",                      
                         "WORKSPACE_ID": "[parameters('AzureSentinelWorkspaceId')]",
                         "SHARED_KEY": "[parameters('AzureSentinelSharedKey')]",
+                        "MAX_SCRIPT_EXEC_TIME_MINUTES": "[parameters('MaxRunTime')]",
```

Comment:
Hello @aaronbeuhring, you have not declared the parameters in the global parameters of the function app
Please declare **MaxRunTime** && **MaximumStorageBlobSize**

Reply:
I'm not sure what you are asking for. The parameters I added to this JSON were not originally in this at all. I followed the same format for the other entries in the properties section. The defaults are set in line 20 and 42 of main.py. ex: MAX_SCRIPT_EXEC_TIME_MINUTES = int(os.environ.get('MAX_SCRIPT_EXEC_TIME_MINUTES', 10)) and MAX_CHUNK_SIZE_MB = int(os.environ.get('MAX_CHUNK_SIZE_MB', 500))